ugandaemr_mobile/SUMMARY.md

```diff
@@ -0,0 +1,10 @@
+# Table of contents
```

Comment:
@solemabrothers This file is not needed anymore

Reply:
Done